userland/libtock/FXOS8700CQ.h

```diff
@@ -0,0 +1,21 @@
+#ifndef _FXOS8700CQ_H
```

Comment:
In general, we're moving to the syntax:

`#pragma once` instead of the `ifndef` style. See `adc.h` for an example.